src/coprocessor/dag/dag.rs

```diff
@@ -103,6 +103,15 @@ impl DAGContext {
                     resp.set_data(data);
                     return Ok(resp);
                 }
+                Err(Error::Eval(err)) => {
+                    let mut resp = Response::new();
+                    let mut sel_resp = SelectResponse::new();
+                    sel_resp.set_error(err);
+                    let data = box_try!(sel_resp.write_to_bytes());
+                    resp.set_data(data);
```

Comment:
Maybe we need to continue, and return the response with all records and all warnings finnally?